src/dini/headers.js

```diff
@@ -0,0 +1,205 @@
+//          - alt: alternate text for the image (<img alt=>), defaults to "Logo" or "Logo 1", "Logo 2", ...
+//            if src is not specified, this is the text of the "logo"
+//          - height: optional height of the logo (<img height=>)
+//          - width: optional width of the logo (<img width=>)
+//          - url: the URI to the organization represented by the logo (target of <a href=>)
+//          - id: optional id for the logo, permits custom CSS (wraps logo in <span id=>)
+//          - each logo element must specify either src or alt
+//  - otherLinks: an array of other links that you might want in the header (e.g., link github, twitter, etc).
+//         Example of usage: [{key: "foo", href:"https://b"}, {key: "bar", href:"https://"}].
+//         Allowed values are:
+//          - key: the key for the <dt> (e.g., "Bug Tracker"). Required.
+//          - value: The value that will appear in the <dd> (e.g., "GitHub"). Optional.
+//          - href: a URL for the value (e.g., "https://foo.com/issues"). Optional.
+//          - class: a string representing CSS classes. Optional.
+//  - license: can be one of the following
+//      - "cc-by-sa"
+//      - "cc-by"
+//      - "cc0"
+import { ISODate, htmlJoinAnd } from "../core/utils.js";
+import headersTmpl from "../w3c/templates/headers.js";
+import { hyperHTML } from "../core/import-maps.js";
+import { pub } from "../core/pubsubhub.js";
+
+export const name = "dini/headers";
+
+const W3CDate = new Intl.DateTimeFormat(["de-DE"], {
+  timeZone: "UTC",
+  year: "numeric",
+  month: "long",
+  day: "2-digit",
+});
+
+const status2text = {
+  unofficial: "Entwurf vom",
+  base: "Spezifikation vom",
+};
+
+/**
+ * @param {*} conf
+ * @param {string} prop
+ * @param {string | number | Date} fallbackDate
+ */
+function validateDateAndRecover(conf, prop, fallbackDate = new Date()) {
+  const date = conf[prop] ? new Date(conf[prop]) : new Date(fallbackDate);
+  // if date is valid
+  if (Number.isFinite(date.valueOf())) {
+    const formattedDate = ISODate.format(date);
+    return new Date(formattedDate);
+  }
+  const msg =
+    `[\`${prop}\`](https://github.com/w3c/respec/wiki/${prop}) ` +
+    `is not a valid date: "${conf[prop]}". Expected format 'YYYY-MM-DD'.`;
+  pub("error", msg);
+  return new Date(ISODate.format(new Date()));
+}
+
+export function run(conf) {
+  conf.isUnofficial = conf.specStatus === "unofficial";
+  conf.isBasic = conf.specStatus === "base";
+  conf.isRegular = !conf.isBasic;
+  if (!conf.specStatus) {
+    pub("error", "Missing required configuration: `specStatus`");
+  }
+  conf.title = document.title || "Kein Titel";
+  if (!conf.subtitle) conf.subtitle = "";
+  conf.publishDate = validateDateAndRecover(
+    conf,
+    "publishDate",
+    document.lastModified
+  );
+  conf.publishYear = conf.publishDate.getUTCFullYear();
+  conf.publishHumanDate = W3CDate.format(conf.publishDate);
+  const peopCheck = function(it) {
```

Comment:
We should probably export this from core/headers.js also and do proper localization on it.